donkeycar/parts/fast_stretch.py

```diff
@@ -49,13 +49,13 @@ def fast_stretch(image, debug=False):
     targetFl = Sl * size
     targetFh = Sh * size
 
-    count = histogram[Xl]
-    while count < targetFl:
+    count = 0
```

Comment:
What you did here is incorrect. Please undo the changes to this file. I can explain why this works the way it does.